.gitignore

```diff
@@ -5,4 +5,7 @@ erl_crash.dump
 *.ez
 
 .idea
-*.iml
\ No newline at end of file
+*.iml
+.elixir_ls
+
+.DS_Store
```

Comment:
Can we remove this?

Since this is a pretty common thing on macs, you can ignore it globally with something like...

```
$ git config --global core.excludesfile ~/.excludes
$ echo .DS_STORE > ~/.excludes

```